### PR TITLE
Blog app: Views

### PR DESCRIPTION
**Pull request by julie-ify**
In this milestone, I implemented the views of the blog app from the sneak peek [wireframes.](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md)

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,27 @@
-<h1>Here is a particular user<h1>
\ No newline at end of file
+<div>
+  <%= render partial: "user_profile", locals: {user: @user} %>
+  <div class="border border-3 border-dark container p-2 my-5 bio">
+    <h3 class="py-4">Bio</h3>
+    <p><%= @user.bio %> </p>
+  </div>
+  <ul class="recent-post">
+    <% @posts.each do |p| %>
```
Comment (SajjadAhmad14) [inline app/views/users/show.html.erb:8]: Please add a `#recent_post` helper method and use it in `users/show.html.erb` to render the 3 most recent posts by user.

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,27 @@
...
+            <% end %>
+            <p><%= p.text%></p>
+            <div>
+              <span>Comments: <%= p.comments.size %></span>
+              <span>Likes: <%= p.comments.size %></span>
+            </div>
+          </div>
+          <div>
+            <ul class="border border-3 border-dark container p-2 mt-1 bio">
+              <% p.comments.each do |c| %>
```
Comment (SajjadAhmad14) [inline app/views/posts/index.html.erb:18]: Please add a `#recent_comments` helper method and use it in `posts/index.html.erb` to render the 5 most recent comments on a post.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Juliana, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
GitHub flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/users/show.html.erb (hunk 1 above)
Comment (julie-ify) [reply app/views/users/show.html.erb:8]: Hi @SajjadAhmad14,
I already have the `recent_post` method in the post model, so are you saying I should move it to the post_helper.rb file ?

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @julie-ify,

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**